server/app/main.py

```
@@ -215,6 +253,90 @@ async def health_check():
+            logger.debug(f"Telegram demo call created successfully - Call ID: {call_result.get('call_id')}")
+        
+        return {
+            "status": "success",
+            "message": "Demo call request received and processed",
+            "call_id": call_result.get("call_id") if call_result else None,
+            "timestamp": datetime.now().isoformat()
+        }
+        
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.14.7)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)